cmd/manager/main.go

```diff
@@ -152,8 +152,8 @@ func main() {
 		os.Exit(1)
 	}
 
-	// watch `openshift-machine-api` namespace along with wmco namespace
-	namespaces := []string{"openshift-machine-api", namespace}
+	// watch `""` (nodes), `openshift-machine-api` (machines), namespaces, along with the wmco namespace
```

Comment:
It is unclear what `""` implies. Please make it `WMCO` given you are editing this line.